js/modules/k6/ws/ws_test.go

```diff
@@ -655,6 +657,7 @@ func TestErrors(t *testing.T) {
 		Dialer: tb.Dialer,
 		Options: lib.Options{
 			SystemTags: &metrics.DefaultSystemTagSet,
+			Throw:      null.BoolFrom(true),
```

Comment:
Can you also set this to `true` in the `newTestState` as I would expect a lot of the other tests depend on this behaviour to report failures. So while they might not be failing now. If we break them - it might be needed for this to be set in order for them to be reported correctly.

Reply:
sounds great